### PR TITLE
BUGZ-1028: Fix qmlToScript getting called on script thread and causing crash

### PR DESCRIPTION
**Pull request by huffman**
https://highfidelity.atlassian.net/browse/BUGZ-1028

### CODE REVIEW

File: interface/src/ui/InteractiveWindow.cpp [hunk 1]
```diff
@@ -188,7 +188,7 @@ InteractiveWindow::InteractiveWindow(const QString& sourceUrl, const QVariantMap
                 QQuickItem* rootItem = _dockWidget->getRootItem();
                 _dockWidget->getQuickView()->rootContext()->setContextProperty(EVENT_BRIDGE_PROPERTY, this);
                 QObject::connect(rootItem, SIGNAL(sendToScript(QVariant)), this, SLOT(qmlToScript(const QVariant&)),
-                                 Qt::QueuedConnection);
+                                 Qt::DirectConnection);
```
Comment (jherico) [inline interface/src/ui/InteractiveWindow.cpp]: This kind of direct connection is in contravention to the normal practices.  You should explicitly add a comment here that `qmlToScript` is specifically intended to be safely callable from any thread, and does it's own special work to ensure that the signal that the script receives gets called on the script thread.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20521/HighFidelity-Beta-PR15975-0178cb4.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20521/HighFidelity-Beta-Interface-PR15975-0178cb4.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20521/HighFidelity-Beta-PR15975-0178cb4.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20521/HighFidelity-Quest-Beta-PR15975-0178cb4.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20521/HighFidelity-Beta-PR15975-0178cb4.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20521/HighFidelity-Beta-Interface-PR15975-0178cb4.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20536/HighFidelity-Beta-PR15975-c4b63ed.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20536/HighFidelity-Quest-Beta-PR15975-c4b63ed.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20536/HighFidelity-Beta-PR15975-c4b63ed.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20536/HighFidelity-Beta-Interface-PR15975-c4b63ed.dmg

**Comment by shanzzam** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20542/HighFidelity-Beta-PR15975-c4b63ed.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20542/HighFidelity-Quest-Beta-PR15975-c4b63ed.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20542/HighFidelity-Beta-PR15975-c4b63ed.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20542/HighFidelity-Beta-Interface-PR15975-c4b63ed.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20542/HighFidelity-Beta-PR15975-c4b63ed.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20542/HighFidelity-Beta-Interface-PR15975-c4b63ed.exe

**Pull request merged**